package.json

```diff
@@ -17,11 +17,11 @@
     "@reduxjs/toolkit": "^2.2.7",
     "dotenv": "^16.4.5",
     "firebase": "^10.12.3",
-    "prop-types": "^15.8.1",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
+    "react-hot-toast": "^2.4.1",
```

Comment:
Please don't use extra packages if it's not needed. For toast you can use MUI snackbar.